circle.yml

```diff
@@ -4,6 +4,18 @@ machine:
   java:
     version: oraclejdk8
 
+dependencies:
+  pre:
+    - wget -q https://dl.bintray.com/sbt/debian/sbt-0.13.16.deb
+    - sudo dpkg -i sbt-0.13.16.deb
+  post:
+    - find ~/.ivy2/cache -name "ivydata-*.properties" -print -delete
```

Comment:
Why do we need this?

Reply:
There is the explanation of this line in the documentation:
> Cleanup the cached directories to avoid unnecessary cache updates
http://www.scala-sbt.org/1.0/docs/Travis-CI-with-sbt.html#Caching